### ISSUE TITLE
pc crashes after lunching any game on vm

### ISSUE THREAD

**Issue by shayanpirayesh86** [jamesstringerparsec/Easy-GPU-PV#228]
i dont know why does it hapen my pc has core i5  12th gen with 32GB ram and gtx1650 
when i run games pc crashes and the stop code is "system thread exception not handled"

**Comment by chunkym0nk3y99** [comment]
What driver version are you on? Any driver version newer then 522.25 is crashing the host for me as well.

**Comment by shayanpirayesh86** [comment]
so you mean if i downgradew my drivers it will work?

**Comment by shayanpirayesh86** [comment]
my driver is the latest version 526.98

**Comment by chunkym0nk3y99** [comment]
Yes, I recommend rolling back to driver version 522.25 or earlier and see if that fixes your problem. Don't forget to update drivers on the VMs also.

**Comment by shayanpirayesh86** [comment]
thanks that worked great!
